snowpack/src/build/build-import-proxy.ts

```diff
@@ -97,6 +97,13 @@ export function wrapHtmlResponse({
     return match;
   });
 
+  // replace any env variables defined in the config
+  for (const envVar in config.env) {
```

Comment:
Can you explain this change? It appears to look for the user of an environment variable in the code and replace it with the actual value. Is there an associated issue with this change?

Reply:
Yeh sure, the issue was that environment variables from the config were not being injected into HTML at all. See here https://github.com/snowpackjs/snowpack/pull/3100/commits/61543aa87ac890928c73921d9fb44fb517fe273c#diff-bde35a1eefc1c53bd3b67174d6675ee71e5ca84eb916abefe548f6025532379dL29

I'm not sure if they were meant to be injected but the docs say that the recommended way to declare env variables now is as part of the config.